pandas/core/indexes/base.py

```diff
@@ -2159,7 +2159,8 @@ def dropna(self, how="any"):
 
         Returns
         -------
-        valid : Index
+        valid
```

Comment:
```suggestion
        Index
```

Can just list the type of object being returned here